### ISSUE TITLE
Spinning Wheel related crash

### ISSUE THREAD

**Issue by Jovahkin** [Um-Mitternacht/Bewitchment#429]
### Expected Behavior
Place three string into the spinning wheel (loom), to craft cobwebs.

### Actual Behavior
Instant Crash

### Steps to Reproduce
Place three string into the spinning wheel (loom), to craft cobwebs.

### Version of Minecraft, Bewitchment, Forge, Patchouli, and Baubles used
1.12.2 MC Forge 1840 (Bewitchment for 1.12.2) Baubles also used.

### Logs, in the event of a crash
Can upload a report later. Using MCedit to remove the loom can fix the issue. :)

**Comment by anqingchen** [comment]
Fixed in 74238c94b0f2f59581079abd4db22cfd549345ef, should be out in the next update/testbuild

**Comment by Nonsanity** [comment]
I just had this very crash with version 0.0.20.8, here's the log:

https://pastebin.com/4tWzG4Py

It creates a permanent crash state, meaning when the chunk containing the spinning wheel is loaded it immediately crashes. (As in our case where that chunk happens to be chunk loaded, so the server can never start at all.)

**Comment by anqingchen** [comment]
Sorry about that, it definitely should be fixed in the next update, releasing soon. Is there anyway to edit out the spinning wheel? 

We might be able to push out a hot fix? @Sunconure11 

sidenote @Nonsanity, I watch your videos lol, totally didnt copy your Avant III base :P

**Comment by Nonsanity** [comment]
I removed the mod, restarted, then reinstalled it. Drastic, but the server is fairly new, so I don't think anyone had started Bewitchment but me. (As I was the one that tried to make the cobwebs.) I've always wished there was a way to remove a specific block on startup—from any mod. I've had to edit world files more times than I care to remember.

Thanks for watching! I should be resuming the channel not too long from now... Hopefully. :)
